draft-ietf-httpbis-connect-tcp.md

```diff
@@ -44,9 +44,9 @@ HTTP/3 uses a UDP transport, so it cannot be forwarded using the pre-existing CO
 
 ## Problems
 
-Classic HTTP CONNECT proxies are identified by an origin.  The proxy does not have a path of its own.  This prevents any origin from hosting multiple distinct proxy services.
+Classic HTTP CONNECT proxies are identified by a host, port, and scheme.  The proxy does not have a path of its own.  This prevents any origin from hosting multiple distinct proxy services.
```

Comment:
Classic HTTP CONNECT proxies don't have a scheme.

```suggestion
Classic HTTP CONNECT proxies are identified by a host and port.  The proxy does not have a path of its own.  This prevents any origin from hosting multiple distinct proxy services.
```

Reply:
Hm. Don't they essentially have a scheme? We have both 'HTTP' and 'HTTPS' CONNECT proxies, so that *seems* like a scheme.

Reply:
Yeah maybe "host, port, and use of TLS"?

Scheme is part of URI, but CONNECT does not deal with an URI.